src/robot_properties_nyu_finger/config.py

```diff
@@ -130,6 +128,132 @@ class NYUFingerConfig(NYUFingerAbstract):
     v0 = zero(robot_model.nv)
     a0 = zero(robot_model.nv)
 
+class NYUFingerConfig0(NYUFingerConfig):
```

Comment:
I propose to rename it to `NYUFingerDoubleConfig0`. This way there is no confusion once we will have a `NYUFingerTrippleConfig0` later on.

Same applies to the URDF naming and the `NYUFingerDoubleConfig1` below.

Reply:
what about DuoFinger, TriFinger, etc...
why config0, 1, etc.

Reply:
The idea was to avoid confusion with the TriFinger robot at MPI. 
The index for config0, 1 etc. is to distinguish each finger in the multi-finger setup as they have different base poses defined by a fixed joint "base_to_finger" in the urdf.

Reply:
I don't understand why don't you create a single urdf for the 3 finger at once?
It would be easier, no?
You can have a base with 3 body linked via FixedJoints and then the fingers attached to each body.
You can even parameterize this using xacro.
Then creating a urdf for 1, 2, 3, N fingers would be easy.

Reply:
It was like this. But in DGHead/SimHead we create one head for each finger and we write controllers for each finger separately with nq=nv=nu=3. If we use a singer URDF for all the robots, then we have nq=nu=nv=3n (n being the number of robots); we will have to additionally match the joint indices with each robot in simulation. This is not ideal as we would like to have minimum change between the code for simulation and for real hardware. 
@jviereck any additional comments?

Reply:
> @jviereck any additional comments?

No.

Reply:
Ok so you have like 3 master boards, 1 per finger and then you control them separately? or you have one executable?
If you have 1 executable then the hardware mimics the fact that you have 1 urdf instead of 3.
But ok if you both think it is the way to go.

Reply:
We use one master board for each finger. We use one executable/python script but spawn one hardware process for each finger.